react-commenting.js

```diff
@@ -73,13 +78,15 @@ class Board extends Component{
   render(){
     const { gameBoard, currentPlayer, winner } = this.state
     // 7) Here:
+    //This return the map of an array with the squares to be rendered.
     let mappedGameBoard = gameBoard.map((value, index) => {
       return(
         <Square
           value={ value }
           index={ index }
           key={ index }
           {/* 8) Here: */}
+          //Function that is used for click.
           gamePlay={ this.gamePlay }
```

Comment:
Yup, this is calling the method gamePlay and passing that method to <Square /> as props.